site/src/components/site-header.md

```diff
@@ -0,0 +1,499 @@
+---
+title: Site header
+description: The Site header component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - site-header
+---
+
+# Site header
+
+Site header component allows adding full width container to hold everything site's header might need - the logo, brand link, navigation, and other controls.
+
+**This page is best viewed with the content section full screened because banners are intended to be full width elements. Go full screen by clicking the button in the top right corner.**
+
+## Usage
+
+The site header is composed of the following elements:
+
+| Component             | Description                                                 |
+| --------------------- | ----------------------------------------------------------- |
+| `site-header`         | The main container for the site header.                     |
+| `site-header-button`  | Buttons within the site header.                             |
+| `site-header-logo`    | The logo displayed in the site header.                      |
+| `site-header-divider` | A vertical divider between elements.                        |
+| `site-header-brand`   | The brand name or title in the site header.                 |
+| `site-header-nav`     | The navigation container within the site header.            |
+| `site-header-panel`   | A panel that can be expanded to display additional content. |
+
+Below is the example of the site header with logo, brand and navigation links.
+
+```html-no-indent
+<div class="site-header">
+	<!-- Microsoft logo -->
+	<a href="https://www.microsoft.com" aria-label="Microsoft">
+		<svg
+			class="site-header-logo"
+			aria-hidden="true"
+			viewBox="0 0 25 25"
+			fill="none"
+			xmlns="http://www.w3.org/2000/svg"
+		>
+			<path d="M11.5216 0.5H0V11.9067H11.5216V0.5Z" fill="#f25022" />
+			<path d="M24.2418 0.5H12.7202V11.9067H24.2418V0.5Z" fill="#7fba00" />
+			<path d="M11.5216 13.0933H0V24.5H11.5216V13.0933Z" fill="#00a4ef" />
+			<path d="M24.2418 13.0933H12.7202V24.5H24.2418V13.0933Z" fill="#ffb900" />
+		</svg>
+	</a>
+
+	<!-- Divider -->
+	<div class="site-header-divider"></div>
+
+	<!-- Brand -->
+	<a href="~/src/index.md" class="site-header-brand">
+		<span>Brand</span>
+	</a>
+
+	<!-- Navigation -->
+	<nav class="site-header-nav">
+		<ul class="display-flex gap-xxs">
+			<li>
+				<a href="#!" class="site-header-button">
+                    <span>Link 1</span>
+                </a>
+			</li>
+			<li>
+				<a href="#!" class="site-header-button">
+                    <span>Link 2</span>
+                </a>
+			</li>
+			<li>
+				<a href="#!" class="site-header-button">
+                    <span>Link 3</span>
+                </a>
+			</li>
+		</ul>
+	</nav>
+</div>
+```
+
+### Logo
+
+There is an atomic available to position the logo right in the middle of header - `site-header-logo-centered`.
```

Comment:
Since this class is paired with one element, it isn't strictly an atomic:
```suggestion
You can use ` site-header-logo-centered` subcomponent class to center the logo.
```